config/rubocop/1.9.1/rubocop-2.7.yml

```diff
@@ -44,6 +44,10 @@ Lint/FlipFlop:
 Metrics/AbcSize:
   Max: 30
 
+Metrics/BlockLength:
+  Exclude:
+    - 'app/admin/**/*'
```

Comment:
I was reading if it was possible to exclude only admin files of AA and not for example, concern files inside `/admin` directory that should have the BlockLength check, and found this SO [post](https://stackoverflow.com/a/40860646). Should we add those paths as well?

Reply:
Updated 8662c4c256d7387bc2473a62c1fc05b87e472148